### PR TITLE
feat: add search

### PR DESCRIPTION
**Pull request by traviscrist**
Screen Shot 2020-06-22 at 8 40 18 AM
Screen Shot 2020-06-22 at 8 39 44 AM

Our config is at: https://github.com/algolia/docsearch-configs/blob/master/configs/webmonetization.json

### CODE REVIEW

**Comment by melissahenderson** [comment]
@traviscrist Should Uphold and GateHub show up?

**Comment by traviscrist** [comment]
> @traviscrist Should Uphold and GateHub show up?

lol, idk it was just a sample to show it's working... lets get this deployed and then we can play with it more.

File: website/siteConfig.js [hunk 1]
```diff
@@ -74,13 +74,18 @@ const siteConfig = {
   // For top-level user or org sites, the organization is still the same.
   // e.g., for the https://JoelMarcey.github.io site, it would be set like...
   //   organizationName: 'JoelMarcey'
-
+  algolia: {
+    apiKey: '522665321749697a7b612bb54dbdb0b4',
```
Comment (adrianhopebailie) [inline website/siteConfig.js:78]: I assume this is not sensitive?

Comment (traviscrist) [reply website/siteConfig.js:78]: double checking with them, I don't think it is.

Comment (traviscrist) [reply website/siteConfig.js:78]: Confirmed, it's meant to be public with Algolia.

Comment (traviscrist) [reply website/siteConfig.js:78]: https://docsearch.algolia.com/docs/faq#can-i-share-the-apikey-in-my-repo

**Pull request merged**